insights/specs/datasources/rsyslog_errorfile.py

```diff
@@ -0,0 +1,40 @@
+"""
+Custom datasource to get the list of the rsyslog errorfile paths.
+"""
+
+from insights.core.context import HostContext
+from insights.core.plugins import datasource
+from insights.combiners.rsyslog_confs import RsyslogAllConf
+from insights.core.exceptions import SkipComponent
+
+
+@datasource(RsyslogAllConf, HostContext)
+def errorfiles(broker):
+    """
+    This datasource provides the list of the rsyslog errorfile paths.
+
+    Sample data returned::
+
+        ['/var/log/omelasticsearch.log', '/var/log/oversized.log']
```

Comment:
Following up on the above comment, here return a string separated by a whitespace, like `/var/log/omelasticsearch.log /var/log/oversized.log`, then the output can be wrapped into one single file.